testing/tests/DevExpress.ui.widgets.dataGrid/dataGrid.tests.js

```diff
@@ -5941,6 +5941,120 @@ QUnit.test("change editing.allowAdding with onCellPrepared and dataSource option
     assert.strictEqual($addRowButton.length, 1, "add row button is rendered");
 });
 
+QUnit.module("optionChanged", {
```

Comment:
Move please these tests from dataGrid tests to columnsController and dataController tests. The dataGrid tests are very hard.